enviroment/auser.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows
```

Comment:
I guess the file name was intended to be `asuser.go`?

Reply:
Yes, you are right.

Reply:
@fbrissi just an observation: `asuser_windows.go` file has the same typo issue. And the function `InitAsUser' is not written correctly on both files.